pkg/authenticator/authenticator.go

```diff
@@ -16,11 +16,14 @@ import (
 	"net/url"
 	"os"
 	"strings"
+	"time"
 
 	"github.com/fullsailor/pkcs7"
 )
 
+var infoLogger = log.New(os.Stdout, "INFO: ", log.LUTC|log.Ldate|log.Ltime|log.Lshortfile)
```

Comment:
You can set the displayed info globally with `SetFlags`. Seems like the kind of thing worth doing for the whole client unless you just have this in for temporary debugging purposes.

```
log.SetFlags(log.LUTC | log.Ldate | log.Ltime | log.Lshortfile)
```

Reply:
Logging is now all in one module and we can't use one set of flags since we have both an error and info prefix :(

I did move this stuff into it's own shared module though.

Reply:
The prefix is the 2nd argument though right? So that can still differ regardless. If we want the same set of flags for info and error then you could set them globally, but maybe we don't.

Reply:
Resolved in DMs